org.jacoco.core.test.validation.scala/src/org/jacoco/core/test/validation/scala/targets/ScalaCaseClassTarget.scala

```diff
@@ -0,0 +1,31 @@
+/*******************************************************************************
+ * Copyright (c) 2009, 2020 Mountainminds GmbH & Co. KG and Contributors
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *    Gergely Fábián - initial API and implementation
+ *
+ *******************************************************************************/
+package org.jacoco.core.test.validation.scala.targets
+
+import org.jacoco.core.test.validation.targets.Stubs.{exec, noexec, nop}
+
+/**
+ * Test target for anonymous functions.
+ */
+object ScalaCaseClassTarget {
+
+  case class Foo(a: Int, b: String, c: Set[String], d: Double) // assertPartlyCovered(31, 0)
```

Comment:
31 uncovered branches are caused by the methods generated by the Scala compiler. My next PR will fix this:

https://github.com/gergelyfabian/jacoco/blob/83c1332614a7652d2991f1be0fb6940b75d88432/org.jacoco.core.test.validation.scala/src/org/jacoco/core/test/validation/scala/targets/ScalaCaseClassTarget.scala#L22

Reply:
With Scala 2.13 it's 36 branches uncovered for a single case class.